### PR TITLE
add perseo-core file to docker configuration

### PR DESCRIPTION
**Pull request by AlvaroVega**
Implements #70

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ FROM centos:6
 WORKDIR /opt
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+ENV PERSEO_FE_URL=172.17.0.1:19090
```
Comment (fgalan) [inline Dockerfile]: This seems to be a kind of hardwiring. We cannot assure that Perseo FE will be listening on that IP/port...

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +15,15 @@ RUN yum update -y && yum install -y wget \
...
-  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo \
+  && echo "# This file should be copied by deployment process" > /etc/perseo-core.properties \
+  && echo "# into /etc/perseo-core.properties, with the appropiate permissions" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# URL for invoking actions when a rule is fired" >>  /etc/perseo-core.properties \
+  && echo "action.url = http://${PERSEO_FE_URL}/actions/do" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# Time in milliseconds (long) to "expire" a "dangling" rule" >>  /etc/perseo-core.properties \
+  && echo "rule.max_age= 60000" >>  /etc/perseo-core.properties
```
Comment (fgalan) [inline Dockerfile]: I understand that lines L26 and L23 should use the same style, i.e. either

```
foo = bar
```

or

```
for=bar
```

File: Dockerfile [hunk 3]
```diff
@@ -14,7 +15,15 @@ RUN yum update -y && yum install -y wget \
...
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
-  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo \
+  && echo "# This file should be copied by deployment process" > /etc/perseo-core.properties \
+  && echo "# into /etc/perseo-core.properties, with the appropiate permissions" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# URL for invoking actions when a rule is fired" >>  /etc/perseo-core.properties \
+  && echo "action.url = http://${PERSEO_FE_URL}/actions/do" >>  /etc/perseo-core.properties \
+  && echo "" >>  /etc/perseo-core.properties \
+  && echo "# Time in milliseconds (long) to "expire" a "dangling" rule" >>  /etc/perseo-core.properties \
```
Comment (fgalan) [inline Dockerfile]: I understand that quotes (") surrounding 'expire' and 'dangling' should be escaped.

File: Dockerfile (hunk 1 above)
Comment (cesarjorgemartinez) [inline Dockerfile]: Not is better define a gw host + perseo fe port env vars?

**Pull request closed without merging**